src/pages/Possibility/KivanSlider.vue

```diff
@@ -0,0 +1,371 @@
+<template>
+	<kv-carousel
+		:autoplay="false"
+		ref="KvCarousel"
+		@change="slideTransition($event)"
+	>
+		<div class="slide-placeholder"></div>
+		<div class="bullet-wrap text-center">
+			<div
```

Comment:
These bullets would be well served as a <button> element. Doing so gives us keyboard navigation out of the box, show the right cursor on hover, and announces to screenreaders better. Then we'll want to add :focus and :hover styling. The other piece for that is that inside of the button we'll need text, since you shouldn't have "empty" buttons. Something like "Go to slide ${index}". That text will have a class that hides it visually, but keeps it on the page for assistive technology, foundation has one, "show-for-sr".